src/App.vue

```diff
@@ -15,26 +15,20 @@ export default {
   data() {
     return {
       mobileMode: false,
+      isAuth: this.$auth0.isAuthenticated,
     };
   },
   created() {
-    setTimeout(() => {
-      if (!this.isAuth && !this.$auth0.isLoading) {
-        this.$store.commit("updateToken", "");
-        this.$nextTick(() => {
-          this.$auth0.logout({ redirect: window.location.origin });
-        });
-        return;
-      }
-    }, 1);
     this.$store.dispatch("loadToken");
     let screenSize = window.matchMedia("(max-width:35rem)");
     if (screenSize.matches) {
       this.mobileMode = true;
     } else this.mobileMode = false;
   },
   mounted() {
-    if (!this.token && !this.$auth0.isLoading) this.$auth0.logout();
+    if (!this.isAuth) {
+      this.$router.push("/");
+    }
```

Comment:
@JatinThakur14 what are all these changes in app.vue? are they intentional?

Reply:
Yes now whenever the user session is expired he will be send back to home page